master/reference/calicoctl/resources/ippool.md

```diff
@@ -42,4 +43,10 @@ spec:
 
 | Field    | Description                 | Accepted Values | Schema  | Default    |
 |----------|-----------------------------|--------------|---------|------------|
-| enabled   | When set to true, ipip encapsulation will be used. | true, false | boolean | true
+| enabled  | When set to true, ipip encapsulation will be used. | true, false | boolean | true |
+| mode     | The IPIP mode defining when IPIP will be used.     | always, cross-subnet | string | always |
+
+(*) Routing of packets using IP in IP will be used when the destination IP address
```

Comment:
Is this `*` suppose to be a footnote for something? I don't see another one to go with this